source/configuration_manual/sieve/managesieve/configuration.rst

```diff
@@ -0,0 +1,265 @@
+.. _sieve_managesieve-configuration:
+
+ManageSieve Configuration
+=========================
+
+**NOTE**: If you have used the Sieve plugin before and you have
+``.dovecot.sieve`` files in user directories, you are advised to **make
+a backup first**. Although the ManageSieve daemon takes care to move
+these files to the Sieve storage before it is substituted with a
+symbolic link, this is not a very well tested operation, meaning that
+there is a possibility that existing Sieve scripts get lost.
+
+The ManageSieve configuration consists of ManageSieve protocol settings
+and :ref:`Sieve interpreter <pigeonhole_sieve_interpreter>`-related
+settings. The Sieve interpreter settings are shared with settings of the
+:ref:`Sieve plugin <sieve>`
+for Dovecot's :ref:`Local Delivery Agent (LDA) <lda>` and :ref:`LMTP <lmtp_server>`.
+First, the ManageSieve protocol settings are outlined and then the
+relevant Sieve settings are described.
+
+Protocol Configuration
+----------------------
+
+Along with all other binaries that Dovecot uses, the managesieve and
+managesieve-login binaries are installed during ``make install`` of the
+:ref:`Pigeonhole <sieve_installation>` package.
+The only thing you need to do to activate the ManageSieve
+protocol support in Dovecot is to add ``sieve`` to the ``protocols=``
+configuration line in your dovecot.conf. The managesieve daemon will
+listen on port 4190 by default. As the implementation of the managesieve
+daemon is largely based on the original IMAP implementation, it is very
+similar in terms of configuration. In addition to most mail daemon
+config settings, the managesieve daemon accepts a few more. The
+following settings can be configured in the ``protocol sieve`` section:
+
+:pigeonhole:ref:`managesieve_max_line_length` = 65536
```

Comment:
i thought this does this?

Reply:
I think Timo meant instead of listing setting items, that are links to individual settings on the pigeonhole page, put one link to `settings/pigeonhole#global-settings`.